### PR TITLE
Feature/login page

### PR DESCRIPTION
**Pull request by NateNeslo**

### CODE REVIEW

**Comment by noydb** [comment]
The commit message here on GitHub should be `KUI-001: etc etc.`

**Comment by noydb** [comment]
Please also check the branching standards. We want to name the branch using the project key + ticket number. Don't worry about changing it now, can do it for next PR 👍 .

so branch for this one should have been: `feature/KUI-1`

File: src/stylesheet/globals.scss [hunk 1]
```diff
@@ -1,6 +1,9 @@
+@import 'mixin';
+@import '00_settings/color';
+
+
```
Comment (noydb) [inline src/stylesheet/globals.scss:4]: delete one blank line please

File: src/stylesheet/globals.scss [hunk 2]
```diff
@@ -1,6 +1,9 @@
+@import 'mixin';
+@import '00_settings/color';
```
Comment (noydb) [inline src/stylesheet/globals.scss:2]: I don't see this package inside stylesheet? Are we gonna add it later? 

I really like that you're adding the ITCSS stuff though, I appreciate the pro-activity you're taking with this project. Good stuff man

**Review by noydb — CHANGES_REQUESTED** [review]

File: src/stylesheet/globals.scss (hunk 2 above)
Comment (NateNeslo) [reply src/stylesheet/globals.scss:2]: Yes, I was working on converting the defaults they've got to SCSS for practice, forgot to remove those lines before pushing

**Review by noydb — APPROVED** [review]

**Pull request merged**